new-dashboard/src/App.jsx

```
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import { CssBaseline, AppBar, Toolbar, Typography, Drawer, List, ListItem, ListItemText, Box, Button } from '@mui/material';
+import CloudConfig from './CloudConfig.jsx';
+
+export default function App() {
+  const [app, setApp] = useState(null);
+  const basePath = window.PARSE_DASHBOARD_PATH || '/';
+
+  useEffect(() => {
+    fetch(basePath + 'parse-dashboard-config.json').then(r => r.json()).then(data => {
+      if (data.apps && data.apps.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)